app/models/campaign.rb

```diff
@@ -619,8 +619,8 @@ def sync_to_campaign_bundle
     return unless campaign_bundle
     self.organization_id = campaign_bundle.organization_id
     self.region_ids = campaign_bundle.region_ids
-    self.start_date = campaign_bundle.start_date
-    self.end_date = campaign_bundle.end_date
+    campaign_bundle.update(start_date: start_date) if start_date < campaign_bundle.start_date
+    campaign_bundle.update(end_date: end_date) if end_date > campaign_bundle.end_date
```

Comment:
I might try to condense this to a single db query. We may also want to set a flag here and only update the bundle on save rather than validate.